packages/multi-test/Cargo.toml

```diff
@@ -20,8 +20,8 @@ backtrace = ["anyhow/backtrace"]
 [dependencies]
 cw-utils = { path = "../../packages/utils", version = "0.13.1" }
 cw-storage-plus = { path = "../../packages/storage-plus", version = "0.13.1"}
-cosmwasm-std = { version = "1.0.0-beta6", features = ["staking"] }
-cosmwasm-storage = { version = "1.0.0-beta6" }
+cosmwasm-std = { version = "1.0.0-beta8", features = ["staking"] }
+cosmwasm-storage = { version = "1.0.0-beta8" }
```

Comment:
I guess we want to do this change for all contracts and packages (simply search for `1.0.0-beta6` in the whole project)

Reply:
Yep. Have done this and made version 13.1 -> 13.2. Just about to push.